### ISSUE TITLE
Setting a Default Bottle for .exe Files in Bottles

### ISSUE THREAD

**Issue by outusuke** [bottlesdevs/Bottles#3426]
When clicking 'Open with Bottles,' the application prompts to select which bottles to run. I would prefer to simply click the .exe file and have it run a default bottle instead.
Is there a way to achieve this?

**Comment by outusuke** [comment]
I made a workaround to archive this by creating a new .desktop with the following

[Desktop Entry]
Name=Bottles default
Comment=Run Windows software files directly
Icon=com.usebottles.bottles
Exec=/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=bottles-cli --file-forwarding com.usebottles.bottles run --bottle my-bottle --executable %f
Terminal=false
NoDisplay=true
Type=Application
Categories=Utility;GNOME;GTK;
StartupNotify=true
StartupWMClass=bottles
MimeType=x-scheme-handler/bottles;application/x-ms-dos-executable;application/x-msi;application/x-ms-shortcut;application/x-wine-extension-msp;
Keywords=wine;windows;
X-GNOME-UsesNotifications=true
X-Flatpak=com.usebottles.bottles

I don't know if it's the best solution but it works

**Comment by orowith2os** [comment]
This might be solvable if Bottles installed desktop files for each bottle, which is doable on recent versions of xdp with the DynamicLauncher portal.

**Comment by ihateSELinux** [comment]
This problem still persists. I don't really want to make a custom .desktop file as I would prefer there to just be a setting to default to a particular bottle when double-clicking programs.

**Comment by valinet** [comment]
> This problem still persists. I don't really want to make a custom .desktop file as I would prefer there to just be a setting to default to a particular bottle when double-clicking programs.

Exactly, such a time waster...

**Comment by InkstainTheBat** [comment]
Agreed, this should be reopened. I was about to ask for the same feature

**Comment by ihateSELinux** [comment]
> Agreed, this should be reopened. I was about to ask for the same feature

May as well open a new issue seeing as commoners don't have permission to reopen issues

**Comment by valinet** [comment]
> > Agreed, this should be reopened. I was about to ask for the same feature
> 
> May as well open a new issue seeing as commoners don't have permission to reopen issues

No, may as well start implementing ourselves if we have nothing to do anyway. From a dev point of view, it is NOT nice at all to be flooded by annoying repetitive issues. One per topic is enough - when they will have time, it will be implemented. In the mean time, again, nothing stops us from implementing it and making a PR.

**Comment by InkstainTheBat** [comment]
Still, is there any way to get anyone to mark this as open until then? Doesn't feel right for a still a still unresolved issue to be marked as "closed" for no good reason
